core/src/main/scala/org/apache/spark/internal/config/ConfigEntry.scala

```diff
@@ -17,6 +17,34 @@
 
 package org.apache.spark.internal.config
 
+// ======================================================================================
+//                       The naming policy of configurations
```

Comment:
Also no big deal but "policy" sounds too strong to me. Wonder if we have a better and softer term such as "The guideline for naming configurations"